WayneSociety_Project/templates/Attractions.html

```diff
@@ -85,6 +85,72 @@ <h5 class="card-title"><b>Fox Theatre</b><br></h5> <!--card-->
                     </div>
                   </div>
 
+                  <div class="card-deck">
```

Comment:
Looks good, i was wondering if you plan on spacing the card so users know which attraction they are currently viewing. (e.g) Detriot Arts, Space, Fox Theatre